src/commands/client/update/index.test.ts

```diff
@@ -0,0 +1,76 @@
+import { afterAll, describe, expect, test } from "bun:test";
+import { FetchCommand, UpdateCommand, UpsertCommand } from "@commands/index";
+import { newHttpClient, range, resetIndexes } from "@utils/test-utils";
+import { sleep } from "bun";
+import { Index } from "../../../../index";
+
+const client = newHttpClient();
+
+describe("UPDATE", () => {
+  afterAll(async () => await resetIndexes());
+
+  test("should update vector metadata", async () => {
+    await new UpsertCommand({
+      id: 1,
+      vector: range(0, 384),
+      metadata: { upstash: "test-simple" },
+    }).exec(client);
+
+    const res = await new UpdateCommand({
+      id: 1,
+      metadata: { upstash: "test-update" },
+    }).exec(client);
+
+    expect(res).toEqual({ updated: 1 });
+
+    sleep(5000);
+
+    const fetchData = await new FetchCommand<{ upstash: string }>([
+      ["1"],
+      { includeMetadata: true },
+    ]).exec(client);
+
+    expect(fetchData[0]?.metadata?.upstash).toBe("test-update");
+  });
+
+  test("successfully updates a batch of vectors", async () => {
```

Comment:
can we check what happens when we can't update succesfully? Like upserting two but this time one of them doesn't exist in db so we get {updated: 1}

Reply:
Excuse my OCD, can we please prefix tests with `test should bla bla` 😂